Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -17,9 +20,9 @@
 #include "Core/IOS/ES/ES.h"
 #include "Core/IOS/IOS.h"
 #include "Core/State.h"
+#include "DiscIO/NANDImporter.h"
```

Comment:
Unneeded include: NANDImporter.h

Reply:
Needed for MSVC compilation.

Reply:
Woops, not sure why I wrote that. Ignore me.